test/userdetails-jmeter.jmx

```diff
@@ -0,0 +1,369 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Yep - figured it would be useful to hang on to, and here seemed as good a place as any.